lib/bn.js

```diff
@@ -1049,6 +1049,7 @@
   // TODO(indutny): it may be reasonable to omit it for users who don't need
   // to work with 256-bit numbers, otherwise it gives 20% improvement for 256-bit
   // multiplication (like elliptic secp256k1).
+  // Code generator: https://gist.github.com/fanatid/1e49dab18efbb10c6380
```

Comment:
What you think about moving generation code to separate file? `util/genCombMulTo.js` for example